scrooge-generator/src/main/resources/javagen/service.java

```diff
@@ -91,3 +95,38 @@ public FinagledService (
   {{>finagleService}}
 {{/finagleServices}}
 }
+
+
+class TFCF<T> extends CompletableFuture<T> {
```

Comment:
sorry about that @benhoyt - I did remove you from the list a few hours ago. I am not sure why you are still notified. I dont see you in the list anymore. Can you try clicking on the "Unsubscribe" option on the right side bar and see if that fixes it.